### ISSUE TITLE
Consider creating a workshop folder first

### ISSUE THREAD

**Issue by bast** [coderefinery/git-intro#424]
For all the subfolders like "recipe" to keep everything together to make it easier to navigate and also remove without affecting rest of the hard disk.

**Comment by bast** [comment]
Should this go into practicals instead of a specific lesson @samumantha?

**Comment by samumantha** [comment]
You mean that in the beginning of the workshop we create suggest to create a  separate folder for the workshop materials?

**Comment by bast** [comment]
Yes, this was a good feedback by a learner (?) to recommend this to avoid that learners create many folders in their $home or Desktop and then it would be harder to clean up.

Just wondering whether it should be recommended as part of this lesson or as part of practical intro?

**Comment by samumantha** [comment]
Yes, let's put it in the intro. 
We could make it a checklist for workshop preparation that we slowly go through during intro? So that people can follow, including the screen setup, folder creation, ...

**Comment by bast** [comment]
Yes. Thanks!
